lib/flex_ui/widgets/ecommerce/product_list_item.dart

```diff
@@ -57,7 +58,10 @@ class ProductListItem extends StatelessWidget {
                         rating: product.averageRating!,
                         size: 20,
                       ),
-                    Text(product.price!.formattedValue),
+                    ProductPrice(
```

Comment:
We may need a null check for this one. I've seen some client implementations where price is not set on a product for specific use cases/reasons (i.e. login required to see pricing).

Reply:
Oh right

Reply:
The products with variants too, they have a price range instead of a price